src/heretic/r_plane.c

```diff
@@ -587,18 +588,20 @@ void R_DrawPlanes(void)
             case 28:
             case 29:           // Scroll_North
                 xsmoothscrolloffset = 0;
-                ysmoothscrolloffset = 0;
+                ysmoothscrolloffset = (crispy->scrolleranims) ? (interpfactor << (pl->special - 25)) -
+                    (((63 - ((leveltime >> 1) & 63)) << (pl->special - 25) & 63) * FRACUNIT) : 0;
                 ds_source = tempSource;
                 break;
             case 20:
             case 21:
             case 22:
             case 23:
             case 24:           // Scroll_East
-                xsmoothscrolloffset = -(interpfactor << (pl->special - 20));
+                // vanilla Heretic always animates eastward scrollers
+                xsmoothscrolloffset = -(interpfactor << (pl->special - 20)) + 
+                    (((63 - ((leveltime >> 1) & 63)) << (pl->special - 20) & 63) * FRACUNIT);
                 ysmoothscrolloffset = 0;
-                ds_source = tempSource + ((63 - ((leveltime >> 1) & 63)) <<
-                                          (pl->special - 20) & 63);
+                ds_source = tempSource;
```

Comment:
Why did you break this up and move the addition from `tempSource` to `xsmoothscrolloffset`?